lib/vtls/openssl.h

```diff
@@ -91,6 +91,13 @@ bool Curl_ossl_cert_status_request(void);
 /* this backend suppots CURLOPT_SSL_CTX_* */
 #define have_curlssl_ssl_ctx 1
 
+/* this backend supports TLS_PSK */
+#if OPENSSL_VERSION_NUMBER >= 0x10000000L
+#ifndef OPENSSL_NO_PSK
+#define have_curlssl_tls_psk 1
+#endif  /* OpenSSL version */
+#endif  /* NO_PSK */
```

Comment:
Thanks @ghedo - I'll look into this (and the doc).
Do you think test 1119 fails due to this?